packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -676,6 +676,25 @@ def _serialize_body_parameter(self, builder: OperationType) -> List[str]:
             retval.append(create_body_call)
         return retval
 
+    def _prepare_params(self, builder: OperationType, is_paging: bool) -> List[str]:
+        if is_paging:
+            return [""]
+        retval = []
+        if builder.parameters.grouped:
+            # request builders don't allow grouped parameters, so we group them before making the call
+            retval.extend(_serialize_grouped_body(builder))
+        if builder.parameters.has_body and builder.parameters.body_parameter.flattened:
+            # unflatten before passing to request builder as well
+            retval.extend(_serialize_flattened_body(builder.parameters.body_parameter))
+        if builder.overloads:
+            # we are only dealing with two overloads. If there are three, we generate an abstract operation
+            retval.extend(self._initialize_overloads(builder))
+        elif builder.parameters.has_body:
+            # non-overloaded body
+            retval.extend(self._create_body_parameter(builder))
```

Comment:
I think we want to be conservative here and basically just say: "If we know there's a body, then we do content_type handling in a local place, otherwise everything else will stay the same". I think it's too risky to do a big paging overhaul when fixing this issue. If you still think we should do a paging cleanup, we can do a separate issue for that, and once we're sure we've tested all of the intricacies, then we can update how we generate paging. But for this bug fix, I think we should just fix what is broken

Reply:
Hi @iscai-msft, I agree with you that we shall be conservative to only fix the broken place. Since the problem happens in `_initialize_overloads` so I add `is_paging` to put it outside the `prepare_request`： 
https://github.com/Azure/autorest.python/blob/6b0605e68f9df422991a30fcdfa0c72b6a34e7f3/packages/autorest.python/autorest/codegen/serializers/builder_serializer.py#L1181-L1189
https://github.com/Azure/autorest.python/blob/6b0605e68f9df422991a30fcdfa0c72b6a34e7f3/packages/autorest.python/autorest/codegen/serializers/builder_serializer.py#L711-L717